config/prod/aacr-interactome-s3website.yaml

```diff
@@ -0,0 +1,19 @@
+# Provision a S3 static website for AACR Interactome
+template_path: managed-s3WebCloudfront.yaml
+stack_name: aacr-interactome-s3website
+parameters:
+  # The department for this resource (i.e. CompOnc, SysBio, Platform, etc..)
+  Department: "CompOnc"
+  # The project for this resource (i.e. NTAP, BSMN, DREAM, etc..)
+  Project: "AACR"
+  # The resource owner's email address
+  OwnerEmail: "xindi.guo@sagebase.org"
+  # Domain name for your website (example.org)
+  DomainName: "sagebase.org"
```

Comment:
Change domain name to sagebionetworks.org

Reply:
cannot make that change doing this.  we will need to create a new website then migrate the data to the new website.